worlds/aquaria/Options.py

```diff
@@ -0,0 +1,144 @@
+"""
+Author: Louis M
+Date: Fri, 15 Mar 2024 18:41:40 +0000
+Description: Manage options in the Aquaria game multiworld randomizer
+"""
+
+from dataclasses import dataclass
+from Options import Toggle, Choice, DeathLink, PerGameCommonOptions, DefaultOnToggle, StartInventoryPool
+
+
+class IngredientRandomizer(Choice):
+    """
+    Randomize Ingredients. Select if the simple ingredients (that does not have
+    a recipe) should be randomized. If 'common_ingredients' is selected, the
+    randomization will exclude the "Red Bulb", "Special Bulb" and "Rukh Egg".
+    """
+    display_name = "Randomize Ingredients"
+    option_off = 0
+    option_common_ingredients = 1
+    option_all_ingredients = 2
+    default = 0
+
+
+class DishRandomizer(Toggle):
+    """Randomize the drop of Dishes (Ingredients with recipe)."""
+    display_name = "Dish Randomizer"
+
+
+class TurtleRandomizer(Choice):
+    """Randomize the transportation turtle."""
+    display_name = "Turtle Randomizer"
+    option_no_turtle_randomization = 0
+    option_randomize_all_turtle = 1
+    option_randomize_turtle_other_than_the_final_one = 2
+    default = 2
+
+
+class EarlyEnergyForm(DefaultOnToggle):
+    """
+    Force the Energy Form to be in a location before leaving the areas around the Home Water.
+    """
+    display_name = "Early Energy Form"
+
+
+class AquarianTranslation(Toggle):
+    """Translate to English the Aquarian scripture in the game."""
+    display_name = "Translate Aquarian"
+
+
+class BigBossesToBeat(Choice):
```

Comment:
Since these are just numerical options, you could make this a class a Range instead of a Choice. Something like:
```
class BigBossesToBeat(Range):
   display_name = "Big bosses to beat"
   range_start = 0
   range_end = 5
   default = 0
```

of course you'll need to import Range from Options too.
Same goes for the MiniBossesToBeat class

Reply:
Interesting. I did not know this type of option existed. Thanks.